www/gatsby-node.js

```diff
@@ -29,8 +29,8 @@ function createCssUtilityClassNodes({ actions, createNodeId, createContentDigest
   // We convert to CSS first since we prefer the real values over tokens.
   const compiledCSS = sass
       .renderSync({
-        file: path.resolve(__dirname, '../scss/edx/utilities-only.scss'),
-        // Resolve tildes the way webpack would in our base npm project 
+        file: path.resolve(__dirname, '../scss/core/utilities-only.scss'),
```

Comment:
I'm seeing that `edx/utilities-only.scss` was deleted in this commit, but not that it was added in core. Was it preexisting file there?

Reply:
Yes it was already there.